app/config.c

```diff
@@ -321,9 +360,17 @@ static void test_selection_menu(void)
         display_enabled(POP_R+12, i, test_list[i].enabled);
     }
 
+    bool tty_update = true;
     bool exit_menu = false;
     while (!exit_menu) {
         bool changed = false;
+
+        if (enable_tty && tty_update) {
+            tty_popup_redraw();
+            tty_update = false;
```

Comment:
This is redundant, because you immediately set tty_update back to true (same in other places).

Reply:
yes! the redraw block used to be at the end of the function :-/